e2e/cypress/support/api/user.js

```diff
@@ -115,6 +115,16 @@ Cypress.Commands.add('apiGetUserByEmail', (email) => {
     });
 });
 
+Cypress.Commands.add('apiGetUserByEmailNoError', (email) => {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/users/email/' + email,
+        failOnStatusCode: false,
```

Comment:
Can we modify the existing `apiGetUserByEmail`? Just pass in `failOnStatusCode` with `true` as default?

`Cypress.Commands.add('apiGetUserByEmail', (email, failOnStatusCode = true) => {...}`